test/htmlrenderer.cpp

```diff
@@ -474,6 +474,26 @@ TEST_CASE("tags still work inside <pre>", "[HtmlRenderer]")
 	REQUIRE(links.size() == 0);
 }
 
+TEST_CASE("line breaks are preserved in tags inside <pre>", "[HtmlRenderer]")
+{
+	HtmlRenderer r;
+
+	const std::string input =
+		"<pre><span>very cool\n\n\n"
```

Comment:
Let's also check that the leading whitespace is ignored:

```suggestion
		"<pre>
		<span>very cool\n\n\n"
```